node/lib/util/merge_util.js

```diff
@@ -230,9 +230,17 @@ exports.mergeSubmodule = co.wrap(function *(metaIndex,
     const isHalfOpened = yield opener.isHalfOpened(subName);
     const forceBare = openOption === SUB_OPEN_OPTION.FORCE_BARE;
     const theirSha = change.newSha;
-    yield fetcher.fetchSha(subRepo, subName, theirSha);
-    if (null !== change.ourSha) {
-        yield fetcher.fetchSha(subRepo, subName, change.ourSha);
+    try {
+        yield fetcher.fetchSha(subRepo, subName, theirSha);
+        if (null !== change.ourSha) {
+            yield fetcher.fetchSha(subRepo, subName, change.ourSha);
+        }
+    } catch (e) {
+        console.error(
+            `Unable to fetch changes in submodule '${subName}', ` +
```

Comment:
Did we say that we were going to put this error to stdout?  I'm worried about parsing stderr with stuff from git fetch mixed in.

Reply:
Use error because it is an error :)